### PR TITLE
Added list component.

### PR DESCRIPTION
**Pull request by rutu14**

### CODE REVIEW

**Review by Shubhamkr2610 — APPROVED** [review]
Looks good to me . you are good to go

File: components/list.css [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+}
+  
+.order1 {
+    list-style-type: decimal-leading-zero;
+}
+  
+.order2 {
+    list-style-type: upper-alpha;
+}
+
```
Comment (Ranjeetk489) [inline components/list.css:20]: maybe class name can be order2-list(feels more descriptive )

File: documentation/list.html [hunk 2]
```diff
@@ -0,0 +1,623 @@
...
+                                <li class="colour-items text4 regular">Coffee</li>
+                                <li class="colour-items text4 regular">Tea</li>
+                                <li class="colour-items text4 regular">Coca Cola</li>
+                            </ul> 
+                        </div> 
+                    </div>
+                    <div id="panel-22" role="tabpanel" aria-labelledby="tab-22" hidden>
+                        <div class="code-syntax">
+                            <script type="text/plain" class="line-numbers language-markup">
+<ul class="unorder1 list-colour" title="List 1">
```
Comment (Ranjeetk489) [inline documentation/list.html:219]: the spacing

**Review by Ranjeetk489 — APPROVED** [review]
Aside from minor refactoring code is fine and good to merge

File: components/list.css (hunk 1 above)
Comment (rutu14) [reply components/list.css:20]: Sure.

File: documentation/list.html (hunk 2 above)
Comment (rutu14) [reply documentation/list.html:219]: This spacing is for syntax highlighting part.

**Pull request merged**